packages/nimble-blazor/NimbleBlazor/Components/NimbleMappingText.razor

```diff
@@ -0,0 +1,7 @@
+@namespace NimbleBlazor
+@typeparam TKey
+
+<nimble-mapping-text
+    key="@Key"
+    text="@Text">
```

Comment:
I think even this component should get an `AddtionalAttributes` parameter.

Reply:
yep, every component is the current direction: https://github.com/ni/nimble/blob/main/packages/nimble-blazor/CONTRIBUTING.md#conventions